beaker-vue/src/components/BeakerCell.vue

```diff
@@ -1,14 +1,13 @@
 <template>
   <div
     class="beaker-cell"
-    :draggable="dragEnabled"
     :class="{
         'drag-target': isCurrentTarget
     }"
-    :onDragstart="handleDragStart"
-    :onDrop="handleDrop"
-    :onDragover="handleDragOver"
-    :onDragleave="handleDragLeave"
+    @dragstart="handleDragStart"
```

Comment:
Could these be moved to the DraggableMarker now? (Why not?)

Reply:
It could, but it works fine as it and has no negative effects.
I had it on the other element at one point, but decided it was a little cleaner to have all the drag event definition next to each other rather than spread out, and the other events all should be on this element.